src/renderers/common/XRManager.js

```diff
@@ -914,8 +973,6 @@ class XRManager extends EventDispatcher {
 
 			this.setFoveation( this.getFoveation() );
 
-			this._referenceSpace = await session.requestReferenceSpace( this.getReferenceSpaceType() );
```

Comment:
FYI: #30902 has moved this line so the non-layers code path uses `requestReferenceSpace()` as well. 

If you revert the PR, I guess we have to copy the line into the non-layer else block.

Reply:
thanks! I added it to the other path.